test/rules/assertions/matchDescription.js

```diff
@@ -41,11 +43,34 @@ export default {
       `,
       errors: [
         {
+          line: 3,
           message: 'JSDoc description does not satisfy the regex pattern.'
         }
       ],
       options: [{
-        matchDescription: '[\u0410-\u042F]+.'
+        matchDescription: '[\u0410-\u042F][\u0410-\u044F]+\\.'
+      }]
+    },
+    {
+      code: `
+          /**
+           * Abc.
+           */
+          function quux () {
+
+          }
+      `,
+      errors: [
+        {
+          line: 3,
+          message: 'JSDoc description does not satisfy the regex pattern.'
+        }
+      ],
+      options: [{
+        tags: {
+          'main description': '[\u0410-\u042F][\u0410-\u044F]+\\.',
```

Comment:
1. Shouldn't it be in camelcase? 
2. Imo it is better to place the `mainDescription` in the top level of the option. Since a) `mainDescription` is not a tag name, b) Putting mainDescription in top-level allows schema validation to prevent typo.

Reply:
I chose "main description" as a special case as users couldn't have a tag with this name as it has a space in it, but I see your point re: allowing schema validation (even though it adds a little extra complexity to the code).

Reply:
Done, thanks!